src/slides/1-demos/chevres/demo/component/styles.css

```
@@ -0,0 +1,23 @@
+html,
+body {
+  height: 100%;
+  background: white;
+  font-family: "Inter", sans-serif;
+}
+
+body {
+  margin: 0;
+  display: grid;
+  place-items: center;
+  text-wrap: pretty;
+  background: rgb(230, 230, 230);
```

pour que ça jure moins avec le fond bleu :
```suggestion
  background: rgb(230, 230, 240);
```